todo_app/templates/index.html

```diff
@@ -12,7 +12,23 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+		{% for todolist in todolists %}
+			<li>{{todolist.id}}        {{todolist.title}}          {{todolist.status}}</li>
+		{% endfor %}
       </ul>
     </div>
   </div>
+  
+  
+  <form action="http://localhost:5000/addtodoitem" method="post" class="add-item">
+	<div class="add-item">
+		<label>Enter the Item Title: </label>
+		<input type="text" name="item_title" required />
```

Comment:
A label can be bound to an input if you put an `id` on the input and a matching `for` on the label.